packages/venia-ui/lib/RootComponents/Product/product.gql.js

```diff
@@ -0,0 +1,300 @@
+                __typename
+                categories {
+                    id
+                    breadcrumbs {
+                        category_id
+                    }
+                }
+                description {
+                    html
+                }
+                id
+                media_gallery_entries {
+                    id
+                    label
+                    position
+                    disabled
+                    file
+                }
+                meta_description
+                name
+                price {
+                    regularPrice {
+                        amount {
+                            currency
+                            value
+                        }
+                    }
+                }
+                sku
+                small_image {
+                    url
+                }
+                url_key
+                ... on ConfigurableProduct {
+                    configurable_options {
+                        attribute_code
+                        attribute_id
+                        id
+                        label
+                        values {
+                            default_label
+                            label
+                            store_label
+                            use_default_value
+                            value_index
+                            swatch_data {
+                                ... on ImageSwatchData {
+                                    thumbnail
+                                }
+                                value
+                            }
+                        }
+                    }
+                    variants {
+                        attributes {
+                            code
+                            value_index
+                        }
+                        product {
+                            id
+                            media_gallery_entries {
+                                id
+                                disabled
+                                file
+                                label
+                                position
+                            }
+                            sku
+                            stock_status
+                            price {
+                                regularPrice {
+                                    amount {
+                                        currency
+                                        value
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+export const PRODUCT_TYPE_POLICIES = {
+    Query: {
+        fields: {
+            product: {
+                read(cachedProduct, { cache, variables }) {
+                    try {
+                        return cache.readFragment({
```

Comment:
It worked before - I just moved it into a custom query rather than having the product talon call it. I'm considering reverting this though as it doesn't add any value aside from being the first custom query written :D

Reply:
Resolving - no longer using prefetch.